lib/components/viewers/viewers.css

```diff
@@ -26,6 +26,11 @@
   animation-name: yellowfade;
 }
 
+/* Route Details Link a11y compatibility */
+a.routeDetails {
```

Comment:
Should that apply to all `a` links?

Reply:
These links only have to be darker because of the non-white background they're on.